docs/bclinux-installation-guide.md

```diff
@@ -0,0 +1,8 @@
+# Installing Intel® Clear Containers 3.0 on BCLinux* version 7
+## Introduction of BC-Linux
+[BC-Linux](https://www.bclinux.org/)（BigCloud Enterprise Linux） is an enterprise Linux operating system developed by China Mobile for cloud computing and big data users.
```

Comment:
Remove extra space: ...(BigCloud Enterprise Linux) is an...

Reply:
OK, There is duplication here and I will remove it. Thanks.